flink-end-to-end-tests/test-scripts/test_kubernetes_embedded_job.sh

```diff
@@ -30,12 +29,13 @@ export FLINK_IMAGE_NAME=test_kubernetes_embedded_job
 export OUTPUT_VOLUME=${TEST_DATA_DIR}/out
 export OUTPUT_FILE=kubernetes_wc_out
 export FLINK_JOB_PARALLELISM=1
-export FLINK_JOB_ARGUMENTS='"--output", "/cache/kubernetes_wc_out"'
+export FLINK_JOB_ARGUMENTS='"--output", "/tmp/kubernetes_wc_out"'
+export FLINK_APPLICATION_NAME="test-application"
 
 function cleanup {
-    kubectl delete job flink-job-cluster
-    kubectl delete service flink-job-cluster
-    kubectl delete deployment flink-task-manager
+    kubectl delete job ${FLINK_APPLICATION_NAME}-master
+    kubectl delete service ${FLINK_APPLICATION_NAME}-master
```

Comment:
I think better to use a suffix (instead of `-master`) that is consistent with the template prefix (currently `job-cluster`).

You may consider something like `job-manager` too, for both template file names and Kubernetes resources.